child_compassion/models/project_lifecycle_event.py

```diff
@@ -114,4 +114,8 @@ def process_commkit(self, commkit_data):
             project = self.create(vals)
             project_ids.append(project.id)
 
+            self.env['compassion.project'].search([
+                ('icp_id', '=', single_data.get('ICP_ID'))
+            ]).status_date = fields.date.today()
```

Comment:
I know you didn't code this but can you change variable name `project` to `lifecycle` to avoid confusion ?
Then instead of searching the project you can directly use the relation `lifecycle.project_id.status_date = fields.Date.today()` (notice the capital D)